update_duktape.sh

```diff
@@ -0,0 +1,14 @@
+SRC_LOCATION=duktape/src/main/jni/
+#Download the things
+wget -O duktape.tar.xz $(curl -s https://api.github.com/repos/svaarala/duktape/releases/latest | grep 'browser_' | cut -d\" -f4) 
+#Extract the release
+mkdir tmp
+tar xvfC duktape.tar.xz tmp/
+#Copy over the new sources
+cp -R tmp/duktape-*/src/ $SRC_LOCATION
+#Patch the duk_config file to use java's timezone offset
+sed -i bak -e 's/^#undef DUK_USE_DATE_GET_LOCAL_TZOFFSET.*$/#define DUK_USE_DATE_GET_LOCAL_TZOFFSET android__get_local_tzoffset/' $SRC_LOCATION/duk_config.h
```

Comment:
You can pass `''` to `-i` to skip the backup which will eliminate the need for the `rm` of it below.